src/apix-components/apix.utils.ts

```diff
@@ -0,0 +1,128 @@
+                // if call successful then can access internal network
+                this.isInternalNetwork0 = true;
+                console.log("internal network");
+            }).catch(response => response);
+  }
+
+  public isInternalNetwork(): boolean {
+    return this.isInternalNetwork0;
+  }
+
+  public static createDisplayStringForProducts(products): string {
+    var productDisplayString = "";
+      // create a display string to be used in the list view
+      if (products && products.length > 0) {
+        productDisplayString = products.join(",").replace(new RegExp(";", 'g')," ");
+      }
+      return productDisplayString;
+  }
+
+  public static createProductListNoVersions(products): string[] {
+    var productListNoVersions = [];
+    for (let product of products) {
+      var productPair = product.split(";");
+      productListNoVersions.push(productPair[0]);
+    }
+    return productListNoVersions;
+  }
+
+  public static formatLocalApis(apis: any[]): any[] {
+    var result: any[] = [];
+    if (apis) {
+        for (var i=0; i<apis.length; i++) {
+            var api = apis[i];
+            api.id = 10000 + i;
+            api.source = "local";
+            if (!api.type || 0 === api.type.length) {
+                if (api.url && api.url.endsWith(".json")) {
+                    api.type = "swagger";
+                } else if (api.url && api.url.endsWith(".raml")) {
+                    api.type = "raml";
+                } else {
+                    api.type = "html";
+                }
+            }
+            // create a display string to be used in the list view
+            api.productDisplayString = this.createDisplayStringForProducts(api.products);
+
+            // remove version numbers from the products on the api for filter purposes
+            api.products = ApixUtils.createProductListNoVersions(api.products);
+
+            // tags
+            var languages = [];
+            if (api.tags && api.tags.length > 0) {
+                for (let tag of api.tags) {
+                    if (tag.category === 'programming-language') {
+                        languages.push(tag.name);
+                    }
+                }
+            }
+            api.languages = languages;
+
+            var apiUrl = "#!/apis/" + api.id;
+
+            // Add api details to search content
+            //api.methods = AppUtils.createMethodsForProduct(value.type, value.url, apiUrl);
+
+            result.push(api);
+        }
+    }
+    return result;
+  }
+
+  /**
+   * This utility function is to work around an issue with insecure certificates on vdc-download.vmware.com.
+   * As it turns on we figured out that in fact the certificate is OK, but this is a bug in many webkit browsers
+   * including Chrome.  For Chrome browsers version 57 or later is needed (63 has the issue).  It seems that it
+   * is also an issue for Safari.
+   */
+  public static fixVMwareDownloadUrl (url): string {
```

Comment:
is this still used, or is this not an issue any more?  it seems that it is stubbed out...

Reply:
No, it is not needed anymore.  I have deleted it.